package/kedro_viz/models/utils.py

```diff
@@ -0,0 +1,25 @@
+"""`kedro_viz.models.utils` contains utility functions used in the `kedro_viz.models` package"""
+
+import logging
+
+from kedro.io import AbstractDataSet
+
+logger = logging.getLogger(__name__)
+
+
+def get_dataset_type(dataset: AbstractDataSet) -> str:
+    """Get the type of a dataset as a string-- the abbreviated name of the module to
+    which ``dataset`` belongs, joined with the name of its class.
+
```

Comment:
Some clarifying examples here would be nice.
```suggestion
# When dataset is an instance of any of the following, the returned type is 
# be "plotly.plotly_dataset.PlotlyDataSet".
# kedro.extras.datasets.plotly.plotly_dataset.PlotlyDataSet
# kedro_datasets.plotly.plotly_dataset.PlotlyDataSet
# my.custom.path.to.plotly.plotly_dataset.PlotlyDataSet
# Any module paths that contain only a package and not a subpackage
# will is as is, e.g. a dataset of type package.PlotlyDataSet gives "package.PlotlyDataSet"

```

There's a comment above `TRACKING_DATASET_GROUPS` in experiment_tracking.py which is now out of date and confusing. Please could you change/remove that also?

Reply:
I agree, some examples would be a really nice addition, I added these in [5c5765d](https://github.com/kedro-org/kedro-viz/pull/1232/commits/5c5765d1b2475327dbad07ab2947ee8abb6226ef). I tried to express the behaviour of the function through examples, since I found it hard to get the wording of descriptions both accurate and clear.

I replaced the out-of-date comment in `experiment_tracking.py` ([6cff8e9](https://github.com/kedro-org/kedro-viz/pull/1232/commits/6cff8e929a8a26914fe080b93645ac1403cbe19c)), let me know if you think the replacement is helpful.

Reply:
This looks great now, nice idea to do it like a doctest rather than my wordy and not very clear text 👍